internal/pkg/config/env_defaults.go

```diff
@@ -40,43 +40,44 @@ const (
 	defaultArtifactMaxBody  = 0
 
 	defaultEnrollInterval = time.Millisecond * 10
-	defaultEnrollBurst    = 100
-	defaultEnrollMax      = 50
```

Comment:
I added it as a comment to the `limit` struct below, having max < burst functionally will result in burst == max and the burst setting is for the `rate.Limiter` and the max setting is for our request controller